### PR TITLE
adding Kanagawa Theme

### PR DESCRIPTION
**Pull request by sspaeti**

### CODE REVIEW

**Comment by Gusted** [comment]
Hi @sspaeti,

Is there an website, repository or document where I can learn more about this color scheme?

**Comment by sspaeti** [comment]
Sure: https://github.com/rebelot/kanagawa.nvim

It's a combination of Gruvbox and Tokyo Night.

File: src/config/color-schemes.drconf [hunk 1]
```diff
@@ -88,3 +88,15 @@ LIGHT
 background: #d5d6db
 text: #343b58
 
+
```
Comment (Gusted) [inline src/config/color-schemes.drconf:91]:
```suggestion
```

**Comment by Gusted** [comment]
Not sure what's going on, Github doesn't like this PR to get merged, will retry in a few hours/days.

image

**Comment by Gusted** [comment]
Really weird...

@sspaeti could you make a new PR for this?

image

**Comment by sspaeti** [comment]
@Gusted I added a new one, let's hope that works 🤞🏻🙂 https://github.com/darkreader/darkreader/pull/10277

**Pull request closed without merging**